File: common/src/main/java/dev/throwouterror/game/common/networking/TextPacket.java
/*
 * Copyright (c) Creepinson
 */

package dev.throwouterror.game.common.networking;

/**
 * @author Creepinson
 */
public class TextPacket extends Packet {
    public TextPacket(String name, String data) {
        super(name, data);
    }

    @Override
    public String getData() {
        return (String) super.getData();
    }
}

File: common/src/main/java/dev/throwouterror/game/common/networking/TextPacket.kt
/*
 * Copyright (c) Creepinson
 */
package dev.throwouterror.game.common.networking

/**
 * @author Creepinson
 */
class TextPacket(name: String, data: String) : Packet(name, data) {
    override fun getPacketData(): String {
        return super.getPacketData() as String
    }
}